Storage/src/Bucket.php

```diff
@@ -977,6 +977,9 @@ public function delete(array $options = [])
      *           [feature documentation](https://cloud.google.com/storage/docs/uniform-bucket-level-access),
      *           as well as
      *           [Should You Use uniform bucket-level access](https://cloud.google.com/storage/docs/uniform-bucket-level-access#should-you-use)
+     *     @type string $iamConfiguration.publicAccessPrevention The bucket's
```

Comment:
Can we provide constants or an enum for the valid values? I'm not sure if that's typical in the PHP libraries.

Reply:
It's not generally typical of the PHP library to provide that unless there is added complexity beyond simply matching the structure of the API request object.